src/main/java/com/educare/unitylend/model/User.java

```diff
@@ -4,22 +4,105 @@
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
-import java.util.Date;
-import java.util.UUID;
+import java.time.LocalDate;
 
 
 @Data
 @NoArgsConstructor
 @AllArgsConstructor
 public class User {
-    private UUID userID;
+    private String userid;
     private String password;
     private String name;
-//    private String email;
-//    private Date dob;
-//    private Integer income;
-//    private String locality;
-//    private String college;
-//    private String company;
-//    private UUID walletID;
+    private String email;
+    private LocalDate dob;
+    private Integer income;
+    private Integer borrowingLimit;
+    private String officename;
+    private String collegeuniversity;
+    private String locality;
+
+    public Integer getIncome() {
+        return income;
+    }
+
+    public String getOfficename() {
```

Comment:
When you add @Data annotation, 
There is no need to add getter and setter methods, Lombok takes care of it.

Reply:
It has been done.